### PR TITLE
 net-misc/networkmanager-wireguard: add a snapshot version

### PR DESCRIPTION
**Pull request by ConiKost**
Since currently only 9999 exists, and last update is from 2018, it makes
sense to add a snapshot version, which can get keywords.

Also bumped to EAPI=7 and taking ownership by myself.

Package-Manager: Portage-2.3.72, Repoman-2.3.17

### CODE REVIEW

File: net-vpn/networkmanager-wireguard/networkmanager-wireguard-20181128.ebuild [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Copyright 1999-2019 Gentoo Authors
```
Comment (jhenkehd) [inline net-vpn/networkmanager-wireguard/networkmanager-wireguard-20181128.ebuild]: The version is problematic for a snapshot of a yet to release software. You should use 0_pre20181128 instead, so any eventual release version is actually considered higher by portage.

Comment (ConiKost) [reply net-vpn/networkmanager-wireguard/networkmanager-wireguard-20181128.ebuild]: Done.

**Comment by leio** [comment]
Wasn't WG more or less integrated into NM proper now; or the separate package is really still a thing that is needed?

**Comment by leio** [comment]
From 1.16 NEWS:
```
* Add support for WireGuard VPN tunnels to NetworkManager. D-Bus API and libnm
  support all options. nmcli supports creating and managing WireGuard profiles,
  with the exception of configuring and showing peers.
```
Though that may miss graphical UI, etc. Does this separate package provide those on top now?

**Comment by leio** [comment]
Or put otherwise - are you sure we shouldn't just last rite `net-misc/networkmanager-wireguard`?

**Comment by ConiKost** [comment]
> Or put otherwise - are you sure we shouldn't just last rite net-misc/networkmanager-wireguard?

Yes. The integrated stuff doesn't have any support for nm-applet. This plugin does. They also don't collide. It's even listed officially as third party on their homepage. So yes, this plugin provides this on top. Works perfectly fine with my WG VPN. Setup via NM-Applet Gui.

**Pull request closed without merging**